tests/test_config/file_fixtures/test.pylintrc

```diff
@@ -3,15 +3,15 @@
 # Default max amount of messages for reporter to display.
 pyta-number-of-messages = 10
 
-# Set whether the default error messages by pylint should be overwritten by python TA's custom messages
+# set whether the default error messages by pylint should be overwritten by python TA's custom messages
```

Comment:
Don't change this file

Reply:
I'm not sure why you marked this comment as resolved; perhaps you meant to make this change, but either forgot or did not commit the change. Either way, please ensure that you double-check your work before resolving my comments, as I don't want to repeat comments.